modules/install-secured-cluster-services-helm-chart.adoc

```diff
@@ -15,6 +15,9 @@ After you configure the `values-public.yaml` and `values-private.yaml` files, in
 * Scanner V4 Indexer: optional for secured clusters when Scanner V4 is installed
 * Scanner V4 DB: optional for secured clusters when Scanner V4 is installed
 
+:FeatureName: Scanner V4
```

Comment:
Should this be indented with the previous bullet? If it applies to the last 2 bullets, maybe combine them and then indent under?